e2etests/testdata/stable/sql_tables/dagre/board.exp.json

```diff
@@ -0,0 +1,396 @@
+          "constraint": "",
+          "reference": ""
+        }
+      ],
+      "label": "products",
+      "fontSize": 20,
+      "fontFamily": "DEFAULT",
+      "language": "",
+      "color": "#0A0F25",
+      "italic": false,
+      "bold": true,
+      "underline": false,
+      "labelWidth": 99,
+      "labelHeight": 36
+    },
+    {
+      "id": "orders",
+      "type": "sql_table",
+      "pos": {
+        "x": 357,
+        "y": 316
+      },
+      "width": 215,
+      "height": 144,
+      "level": 1,
+      "opacity": 1,
+      "strokeDash": 0,
+      "strokeWidth": 2,
+      "borderRadius": 0,
+      "fill": "#FFFFFF",
+      "stroke": "#0A0F25",
+      "shadow": false,
+      "3d": false,
+      "multiple": false,
+      "tooltip": "",
+      "link": "",
+      "icon": null,
+      "iconPosition": "",
+      "fields": null,
+      "methods": null,
+      "columns": [
+        {
+          "name": "id",
+          "type": "int",
+          "constraint": "",
+          "reference": ""
+        },
+        {
+          "name": "user_id",
+          "type": "int",
+          "constraint": "",
+          "reference": ""
+        },
+        {
+          "name": "product_id",
+          "type": "int",
+          "constraint": "",
+          "reference": ""
+        }
+      ],
+      "label": "orders",
+      "fontSize": 20,
+      "fontFamily": "DEFAULT",
+      "language": "",
+      "color": "#0A0F25",
+      "italic": false,
+      "bold": true,
+      "underline": false,
+      "labelWidth": 74,
+      "labelHeight": 36
+    },
+    {
+      "id": "shipments",
+      "type": "sql_table",
+      "pos": {
+        "x": 669,
+        "y": 18
+      },
+      "width": 293,
+      "height": 180,
+      "level": 1,
+      "opacity": 1,
+      "strokeDash": 0,
+      "strokeWidth": 2,
+      "borderRadius": 0,
+      "fill": "#FFFFFF",
+      "stroke": "#0A0F25",
+      "shadow": false,
+      "3d": false,
+      "multiple": false,
+      "tooltip": "",
+      "link": "",
+      "icon": null,
+      "iconPosition": "",
+      "fields": null,
+      "methods": null,
+      "columns": [
+        {
+          "name": "id",
+          "type": "int",
+          "constraint": "",
+          "reference": ""
+        },
+        {
+          "name": "order_id",
+          "type": "int",
+          "constraint": "",
+          "reference": "orders.id"
+        },
+        {
+          "name": "tracking_number",
+          "type": "string",
+          "constraint": "",
+          "reference": ""
+        },
+        {
+          "name": "status",
+          "type": "string",
+          "constraint": "",
+          "reference": ""
+        }
+      ],
+      "label": "shipments",
+      "fontSize": 20,
+      "fontFamily": "DEFAULT",
+      "language": "",
+      "color": "#0A0F25",
+      "italic": false,
+      "bold": true,
+      "underline": false,
+      "labelWidth": 116,
+      "labelHeight": 36
+    }
+  ],
+  "connections": [
```

Comment:
this test doesn't have the variable?

a d2compiler/compile_test.go entry is appropriate here that checks value of field

Reply:
Because I never added these fields to the exporter.
For a moment I thought I was delusional that everything was working by chance 😂

Reply:
i don't actually think this field should be exported. it's just for the layout step

Reply:
so, removing then 😅

Reply:
yes, this test wasn't the right one to test this change. but i think u should keep it around anyway, cuz we don't have a many to 1 sql table e2e test